### PR TITLE
social login integration

### PR DESCRIPTION
**Pull request by ks-rahul**
1. facebook login
2. google login
3. github login 
4. twitter login (without keys)
5. LinkedIn login
6. contry code list added on registration page phone number

### CODE REVIEW

File: src/components/ComponentPages/Registration/UI/countryCodes.ts [hunk 1]
```diff
@@ -0,0 +1,1212 @@
+export const country = [
```
Comment (akuks) [inline src/components/ComponentPages/Registration/UI/countryCodes.ts]: I think we should get this from the database it self.

File: src/network/services/auth/index.ts [hunk 2]
```diff
@@ -129,3 +129,63 @@ export const verifyOtp = (values: object) => {
...
+  try {
+    const authToken = await createToken();
+
+    const res = await NetworkCall.fetch(
+      UserRequest.userSocialLogin(values, authToken.access_token)
+    );
+
+    return res;
+  } catch (err) {
+    console.log("socialLogin", err.error.data);
```
Comment (akuks) [inline src/network/services/auth/index.ts]: Can we use console.error()?

**Review by akuks — CHANGES_REQUESTED** [review]
please go through the comment

File: src/components/ComponentPages/Registration/UI/countryCodes.ts (hunk 1 above)
Comment (ks-rahul) [reply src/components/ComponentPages/Registration/UI/countryCodes.ts]: country list API implemented

File: src/network/services/auth/index.ts (hunk 2 above)
Comment (ks-rahul) [reply src/network/services/auth/index.ts]: console.log changed to console.error

File: src/components/ComponentPages/Registration/UI/otp.tsx [hunk 3]
```diff
@@ -51,10 +51,6 @@ export default function OTPVerify({ form, onFinish, isModal, closeModal }) {
                 required: true,
                 message: messages.validations.otp,
               },
```
Comment (akuks) [inline src/components/ComponentPages/Registration/UI/otp.tsx]: Move Validation into a separate folder

File: src/components/ComponentPages/Registration/UI/index.tsx [hunk 4]
```diff
@@ -176,10 +175,6 @@ function RegistrationUi({
                 name="email"
                 label={messages.labels.email}
                 rules={[
-                  {
-                    type: "email",
-                    message: messages.validations.email,
-                  },
                   {
                     max: 255,
                     message: messages.validations.middleName,
```
Comment (akuks) [inline src/components/ComponentPages/Registration/UI/index.tsx]: Please move it to validation.

**Review by akuks — CHANGES_REQUESTED** [review]

File: src/components/ComponentPages/Registration/UI/otp.tsx (hunk 3 above)
Comment (ks-rahul) [reply src/components/ComponentPages/Registration/UI/otp.tsx]: moved

File: src/components/ComponentPages/Registration/UI/index.tsx (hunk 4 above)
Comment (ks-rahul) [reply src/components/ComponentPages/Registration/UI/index.tsx]: moved

**Pull request merged**